packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardLarge.tsx

```diff
@@ -0,0 +1,115 @@
+import { CalendarTodayTwoTone } from '@mui/icons-material'
+import { Box, Button, Divider, Link, Stack, Typography } from '@mui/material'
+import React from 'react'
+import { ReleaseCardProps } from './ReleaseCard'
+import { ReleaseCardLargeConfig, ReleaseCardStat } from './ReleaseCardTypes'
+import {
+  createButtonToExploreDataPathAndQueryString,
+  formatReleaseCardData,
+} from './ReleaseCardUtils'
+
+const StatFlex: React.FunctionComponent<ReleaseCardStat> = ({
+  value,
+  label,
+}: ReleaseCardStat) => {
+  return (
+    <Stack
+      direction="column"
+      alignItems="center"
+      padding="20px 40px 20px 40px"
+      gap="10px"
+      width="200px" // ensure that stats are aligned when multiple cards are stacked vertically
```

Comment:
Using a set width ensures that the statistics are aligned across a list of large release cards. Could consider using grid/subgrid instead, but this would be a bit more involved, so I was planning to hold off on until we need to display statistics with widely varying widths so using a set 200px width wouldn't work well.